lib/sources.nix

```diff
@@ -140,7 +140,7 @@ let
       origSrc = if isFiltered then src.origSrc else src;
     in lib.cleanSourceWith {
       filter = (path: type:
-        let relPath = lib.removePrefix (toString origSrc + "/") (toString path);
+        let relPath = lib.removePrefix (origSrc + "/") (path);
```

Comment:
??

`removePrefix` doesn't work on paths, does it? Won't this copy every single `path` to the store?